docs/setup/xml-config.md

```diff
@@ -14,6 +14,13 @@ export RMW_IMPLEMENTATION=name-of-the-rmw
 ```
 Currently, the only supported RMW implementations are: `rmw_cyclonedds_cpp` and `rmw_fastrtps_cpp`.  See [here](https://docs.ros.org/en/rolling/Concepts/About-Different-Middleware-Vendors.html) for more info on the different RMW vendors.
 
+If you have already set your default RMW implementation via the ROS2 setup process and would like to change your default RMW, use 
+
+```sh
+nano ~/.bashrc
```

Comment:
nano is not included in some lite OS distributions and it should be installed.
We should add
```
sudo apt-get update && sudo apt-get install nano
```
To make sure that the next line doesn't fail